third_party/tutorial.proto

```diff
@@ -59,14 +59,21 @@ message Link {
 
 message InlineContent {
   oneof content {
-    string text = 1;
-    string strong_text = 2;
-    string emphasized_text = 3;
+    StylizedText text = 6;
     string inline_code_text = 4;
     Link link = 5;
   }
+
+  reserved 1, 2, 3; 
+  reserved "strong_text" "emphasized_text";
```

Comment:
formatting issue: this was submitted before the build system was in place and error checking this file's format